lib/ZnapZend/ZFS.pm

```diff
@@ -125,6 +125,7 @@ sub snapshotExists {
 sub listDataSets {
     my $self = shift;
     my $remote = shift;
+    my @dataSets;
 
     my @ssh = $self->$buildRemote($remote, [@{$self->priv}, qw(zfs list -H -o name)]);
```

Comment:
oh, my bad, thought `-t` only works w/ `snapshot|volume`

will patch the patch :D